lib/whitehall_importer/import.rb

```diff
@@ -41,10 +41,12 @@ def current?(edition)
 
     def create_users(users)
       users.each_with_object({}) do |user, memo|
-        user_keys = %w[uid name email organisation_slug organisation_content_id]
-        attributes = user.slice(*user_keys).merge("permissions" => [])
-        user_object = User.find_by(uid: user["uid"]) || User.create!(attributes)
-        memo[user["id"]] = user_object.id
+        if user["uid"].present?
+          user_keys = %w[uid name email organisation_slug organisation_content_id]
+          attributes = user.slice(*user_keys).merge("permissions" => [])
+          user_object = User.find_by(uid: user["uid"]) || User.create!(attributes)
+          memo[user["id"]] = user_object.id
+        end
```

Comment:
A guard clause might be a nicer way to approach this with:

```
next unless user["uid"].present?

user_keys = %w[uid name email organisation_slug organisation_content_id]
...
```

Reply:
Thanks for the quick review and the comments!

Just an FYI on this, Rubocop isn't happy with this and prefers the below instead...

```
next if user["uid"].blank?

user_keys = %w[uid name email organisation_slug organisation_content_id]
...
```
So, will use this to get a clean build on CI if that's ok?

Reply:
That's cool - rubocop's suggestion sounds better 👍